package.json

```diff
@@ -44,11 +44,13 @@
   "devDependencies": {
     "@types/chai": "3.4.22-alpha",
     "@types/mocha": "2.2.21-alpha",
+    "@types/mock-fs": "3.6.22-alpha",
```

Comment:
I was wondering if the type definitions need to have the same version as the main module but I took example in the rest. (I'm new to TypeScript)